game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -523,14 +519,6 @@ public IServerMessenger getMessenger() {
     return new LinkedHashMap<>(playerNamesAndAlliancesInTurnOrder);
   }
 
-  @Override
-  public void messengerInvalid(final Throwable reason) {
-    if (ui != null) {
-      JOptionPane.showMessageDialog(ui, "Connection lost", "Error", JOptionPane.ERROR_MESSAGE);
-    }
-    serverSetupModel.showSelectType();
-  }
-
```

Comment:
Great to see changes like this. Was ServerModel simply never used as IMessengerErrorListener?

Reply:
Whether for better or worse, yes, the operations were always a no-op and this was dead code.
It's a possible bug really that this stuff wound up being a no-op. I suspect there was more code thrown at the problem to make the system work properly.